examples/braket_features/create_custom_local_simulator.ipynb

```diff
@@ -0,0 +1,684 @@
+{
```

Comment:
<p>No need to use a bolded subheading for the the description ("In this example, ..."); just use normal text.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/297/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>